suite-native/atoms/src/Modal/BottomSheet.tsx

```diff
@@ -9,10 +9,10 @@ import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
```

Comment:
Wrong folder, it's not Modal anymore.

Reply:
https://github.com/trezor/trezor-suite/pull/5818/commits/3416fa2df587cb76f29b2920a2c66bd5ac97c442